### DISCUSSION TITLE
github pages

### DISCUSSION THREAD

**Discussion by Alwin8** [community/community#122962] [New to GitHub]
can i host porfolios of my freelance clients in github pages

**Comment by gil-son** [comment]
Hello how are you today?

If you check the rules of [GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)

you will find this stretch:

' You can create GitHub Pages sites that are publicly available on the internet. Organizations that use GitHub Enterprise Cloud can also publish sites privately by managing access control for the site. '

In my vision you can public in your host some projects about your freelance and it involves with your technological skills 

If I supported you consider to mark my answer. Good Luck! God bless you!
